### ISSUE TITLE
Cannot run it on my phone

### ISSUE THREAD

**Issue by changhongkuo** [hollance/YOLO-CoreML-MPSNNGraph#50]
TinyYOLO-CoreML only built successfully using simulator but I got error when I tried it on my iPhone6s. My Xcode version is 10 and my iphone has ios 12. I got two error messages:
1. 
Showing Recent Messages
Build system information
error: No account for team "MGCEKGP4Y4". Add a new account in the Accounts preference pane or verify that your accounts have valid credentials. (in target 'TinyYOLO-CoreML')
2. 

Showing Recent Messages
:-1: No profiles for 'net.machinethink.TinyYOLO-CoreML' were found: Xcode couldn't find any iOS App Development provisioning profiles matching 'net.machinethink.TinyYOLO-CoreML'. (in target 'TinyYOLO-CoreML')

Build failed. How do I fix this? Thank you

**Comment by hollance** [comment]
You need to choose your own development account in the Project Settings screen in Xcode. Right now it is set to my own account.

**Comment by changhongkuo** [comment]
Thank you for your help! I solved the problem. I tested your app and it seems like it only detects correctly when you hold the phone vertically instead of horizontally.

**Comment by hollance** [comment]
Yes, you need to rotate the images if you’re holding the phone horizontally.
